### ISSUE TITLE
Contrast of menu icons insufficient accessibility

### ISSUE THREAD

**Issue by clarkepaul** [silverstripe/silverstripe-framework#6408]
Icons are actually black with a 70% opacity which fails on contrast. It is time anyway to replace these with svg icons anyway :).
Would be good to look at also adding icons for the main modules at the same time to pre-empt their updates.
![image](https://cloud.githubusercontent.com/assets/555033/21250084/5e5a2c9a-c3a7-11e6-9c9f-e6c4f2489722.png)

**Comment by dhensby** [comment]
From an accessibility point of view, the icons are unimportant, the contrast of the text is surely what matters (and looks to be the same?!)

It is acceptable to provide alternative devices (the menu text) for accessibility rather than ensure decorative elements are accessible.

**Comment by clarkepaul** [comment]
When the menu is collapsed the icons should be a suitable contrast as they act like text, the text is still accessible by screen readers when the menu is collapsed. If we swap the PNG's with SVG the colour will be more flexible to change with UI.

I guess me raising this issue is partly about updating the icons for SS4, there are only a few contrast issues and my aim would be reduce them.

**Comment by sminnee** [comment]
Not critical for beta1 – specifically, user-testing can be carried out without this.

**Comment by clarkepaul** [comment]
The icons have all been updated to svg fonts so now the colour of the menu **icons and text** can both be updated to #406880 which have a decent contrast, should be able to use the following to produce this colour:
color: darken($base-menu-bg, 56%);

**Comment by clarkepaul** [comment]
I've created a custom set of svg icons which are suitable size-wise to be displayed at 20px for the menu, so please request them from @clarkepaul if someone picks this issue up (not supplied here because they are likely to have additional icons added in the coming months).

Three CSS changes would be required for new icon sizes:
<img width="132" alt="pasted_image_6_06_17__6_13_pm" src="https://cloud.githubusercontent.com/assets/555033/26816074/fc0081ba-4ae3-11e7-8335-9fa1671ad51d.png">

**Comment by GuySartorelli** [comment]
Closed in favour of https://github.com/silverstripe/silverstripe-admin/issues/1970
